tests/pre_merge/tools/conftest.py

```diff
@@ -0,0 +1,56 @@
+"""Fixtures for the tools tests."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+from __future__ import annotations
+
+from tempfile import TemporaryDirectory
+from typing import Generator
+
+import albumentations as A
+import cv2
+import numpy as np
+import pytest
+from albumentations.pytorch import ToTensorV2
+
+from anomalib.config import get_configurable_parameters
+
+
+@pytest.fixture(scope="package")
+def project_path() -> Generator[str, None, None]:
```

Comment:
How about using pytest's builtin `tmp_path` or `tmp_path_factory` fixtures?

Reply:
It does not automatically delete the temporary directory. I have to use `shutil.rmtree` otherwise. Something like 
```python
yield str(tmp_path_factory.mktemp("project"))
    shutil.rmtree(tmp_path_factory.getbasetemp())
```